.gitignore

```diff
@@ -5,7 +5,6 @@
 /build
 /captures
 .externalNativeBuild
-/gradle.properties
```

Comment:
We are always compiling the debug so please add the line to the debug-gradle.properties